src/lib/components/composed/search/FilterEditor.svelte

```diff
@@ -20,7 +21,8 @@
     forceApplyValid: boolean = false,
     editFilterMode: 'one-edit' | 'multi-edit' = 'one-edit',
     tmpFilter: Filter | undefined = undefined,
-    mobile: boolean = false
+    mobile: boolean = false,
+    textfieldBorderRadius = "5px";
```

Comment:
Questa va usata una variabile css, non ci passiamo più le props con lo stile

Reply:
Anzi, secondo me puoi evitare di passartela, se tu dal componente in cbam fai una cosa del genere:

```Svelte
<Component
  ...
  --simple-textfield-border-radius="5px"
></Component>
```

ottieni il risultato voluto.

Reply:
Nuovo commit